lib/internal/process/promises.js

```diff
@@ -50,9 +51,7 @@ function emitWarning(uid, reason) {
     } else {
       process.emitWarning(safeToString(reason), unhandledRejectionErrName);
     }
-  } catch (e) {
-    // ignored
-  }
+  } catch (ignored) {}
```

Comment:
With V8 6.6 we can use [optional catch binding](https://github.com/tc39/proposal-optional-catch-binding), if you want.